README.md

```diff
@@ -0,0 +1,48 @@
+# GeoServer Workflow
+This script automates the process for preparing GeoTIFFs for use in WMS applications and adding them to GeoServer.
```

Comment:
Is it worth adding links (or versions, or anything else?) to geotiff or geoserver - or are those things that will be obvious to anyone using this?